.github/workflows/check-release.yml

```diff
@@ -1,40 +1,28 @@
 name: Check Release
-
 on:
   push:
-    branches: main
+    branches: ['main']
   pull_request:
-    branches: '*'
-
-permissions:
-  contents: write
+    branches: ['*']
 
 jobs:
   check_release:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        group: [check_release]
-        python-version: ["3.9"]
-        node-version: ["18.x"]
-
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-
       - name: Base Setup
         uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
-
+      - name: Install Dependencies
+        run: |
+          pip install .
       - name: Check Release
-        if: ${{ matrix.group == 'check_release' }}
         uses: jupyter-server/jupyter_releaser/.github/actions/check-release@v2
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          version_spec: next
```

Comment:
Could this be the reason for the failure? We probably need to keep this version_spec value?